ArduCopter/mode.h

```diff
@@ -550,9 +550,8 @@ class ModeAutoTune : public Mode {
     bool allows_arming(bool from_gcs) const override { return false; }
     bool is_autopilot() const override { return false; }
 
-    void save_tuning_gains();
+    void save_tuning_gains(bool save);
```

Comment:
Wait, you've got a `save_tuning_gains` which takes a `bool save`?  Ewww.

How about renaming to, `end_autotune(bool save)` ?

Reply:
just went with `disarmed(bool save)` your right  `save_tuning_gains ` was dumb